pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -458,98 +272,7 @@ func TestGetChildObjectsFromPRStatusForTaskNames(t *testing.T) {
-					"r1": {PipelineTaskName: "run-1"},
-				},
-			}},
-			expectedTRNames:        nil,
-			expectedCustomRunNames: []string{"r1"},
-			hasError:               false,
-		}, {
-			name:           "taskrun and run, full embedded",
-			embeddedStatus: config.FullEmbeddedStatus,
-			prStatus: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				TaskRuns: map[string]*v1beta1.PipelineRunTaskRunStatus{
-					"t1": {PipelineTaskName: "task-1"},
-				},
-				Runs: map[string]*v1beta1.PipelineRunRunStatus{
-					"r1": {PipelineTaskName: "run-1"},
-				},
-			}},
-			expectedTRNames:        []string{"t1"},
-			expectedCustomRunNames: []string{"r1"},
-			hasError:               false,
-		}, {
-			name:           "taskrun and run, full embedded, just want taskrun",
-			embeddedStatus: config.FullEmbeddedStatus,
-			prStatus: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				TaskRuns: map[string]*v1beta1.PipelineRunTaskRunStatus{
-					"t1": {PipelineTaskName: "task-1"},
-				},
-				Runs: map[string]*v1beta1.PipelineRunRunStatus{
-					"r1": {PipelineTaskName: "run-1"},
-				},
-			}},
-			taskNames:        sets.NewString("task-1"),
-			expectedTRNames:  []string{"t1"},
-			expectedRunNames: nil,
-			hasError:         false,
-		}, {
-			name:           "full embedded",
-			embeddedStatus: config.FullEmbeddedStatus,
-			prStatus: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				TaskRuns: map[string]*v1beta1.PipelineRunTaskRunStatus{
-					"t1": {PipelineTaskName: "task-1"},
-				},
-				ChildReferences: []v1beta1.ChildStatusReference{{
-					TypeMeta: runtime.TypeMeta{
-						APIVersion: v1alpha1.SchemeGroupVersion.String(),
-						Kind:       pipeline.RunControllerName,
-					},
-					Name:             "r1",
-					PipelineTaskName: "run-1",
-				}},
-			}},
-			expectedTRNames:  []string{"t1"},
-			expectedRunNames: nil,
-			hasError:         false,
-		}, {
-			name:           "both embedded",
-			embeddedStatus: config.BothEmbeddedStatus,
-			prStatus: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				TaskRuns: map[string]*v1beta1.PipelineRunTaskRunStatus{
-					"t1": {PipelineTaskName: "task-1"},
-				},
-				ChildReferences: []v1beta1.ChildStatusReference{{
-					TypeMeta: runtime.TypeMeta{
-						APIVersion: v1alpha1.SchemeGroupVersion.String(),
-						Kind:       pipeline.RunControllerName,
-					},
-					Name:             "r1",
-					PipelineTaskName: "run-1",
-				}},
-			}},
-			expectedTRNames:  nil,
-			expectedRunNames: []string{"r1"},
-			hasError:         false,
-		}, {
-			name:           "default minimal embedded",
-			embeddedStatus: config.MinimalEmbeddedStatus,
+			name: "default",
 			prStatus: v1beta1.PipelineRunStatus{PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
 				TaskRuns: map[string]*v1beta1.PipelineRunTaskRunStatus{
```

Comment:
No tests should have pipelineruns with taskruns in the status anymore. This test name should also be updated to reflect what it's testing

Reply:
Thanks, updated.